src/v/raft/vote_stm.cc

```diff
@@ -265,6 +265,8 @@ ss::future<> vote_stm::update_vote_state(ss::semaphore_units<> u) {
                 "{} - {} ",
                 ec.value(),
                 ec.message());
+          } else {
+              vlog(_ctxlog.info, "replicated configuration");
```

Comment:
This seems like more of a "debug" thing than an "info" thing?  I'm not sure it's going to be meaningful to users.  Unless maybe we rephrase the log messages so that the one line in 254 says "becoming" leader, and this one says "became" leader, to indicate the phased nature of it.

Also maybe this should have some context in the message, at least the term?